index.d.ts

```diff
@@ -24,6 +24,15 @@ declare namespace elementReady {
 		readonly stopOnDomReady?: boolean;
 	}
 
+  type Stoppable = {
+    /**
+     Stop checking for new elements.
+
+     Calling it multiple times does nothing.
+     */
+    stop(): void;
+  };
+
 	type StoppablePromise<T> = Promise<T> & {
```

Comment:
Perhaps

```ts
 	type StoppablePromise<T> = Promise<T> & Stoppable;
```

Reply:
I actually wanted to do so, but I didn't want to change the doc comments.